code/modules/projectiles/projectile/ego_bullets/aleph.dm

```diff
@@ -9,10 +9,10 @@
 	name = "slime projectile"
 	icon_state = "slime"
 	desc = "A glob of infectious slime. It's going for your heart."
-	damage = 88
+	damage = 100
```

Comment:
See below. It does damage over time, so with your buff it will be ~150 black damage.

Reply:
I'm not saying it should be nerfed, but I'm not sure if that was your intent here.

Reply:
Eh I saw the impact damage of ego_magicbullet (80 Black) while having a better firerate/projectile speed but yeah I can revert the damage buff if needed

Reply:
Wasn't needed, but I'll see how the weapon performs in the game.

Reply:
I revert it, I will leave it to someone that eventually do a ego weapon rebalance better have one big pr like the armor one